src/Standard/TIP-4/4.md

```diff
@@ -84,39 +84,49 @@ Every `Storage` contract must implement `TIP4_4Storage`
 pragma ton-solidity >= 0.58.0;
 
 interface TIP4_4Storage {
-    function fill(uint8 id, bytes chunk, address gasReceiver) external;
+    function fill(uint32 id, bytes chunk) external;
     function getInfo() external view responsible returns (
         address nft,
         address collection,
         string mimeType,
-        mapping(uint8 => bytes) content
+        bool zstdEncoding,
```

Comment:
I think `bool` value is wrong. Compression method may differ from zstd. In my opinion, a good solution would be to use a `string`, as is done in [HTTP header Content-Encoding](https://developer.mozilla.org/ru/docs/Web/HTTP/Headers/Content-Encoding)